src/rebar_prv_common_test.erl

```diff
@@ -155,86 +243,39 @@ parse_term(String) ->
 
 create_dirs(Opts) ->
     LogDir = proplists:get_value(logdir, Opts),
-    TestDir = proplists:get_value(dir, Opts),
-    ensure_logdir(LogDir),
-    ensure_testdir(TestDir),
+    ensure_dir([LogDir]),
     ok.
 
-ensure_logdir(Logdir) ->
-    case ec_file:is_dir(Logdir) of
+ensure_dir([]) -> ok;
+ensure_dir([Dir|Rest]) ->
+    case ec_file:is_dir(Dir) of
         true ->
             ok;
         false ->
-            ec_file:mkdir_path(Logdir)
-    end.
+            ec_file:mkdir_path(Dir)
+    end,
+    ensure_dir(Rest).
 
-ensure_testdir(Testdir) ->
-    case ec_file:is_dir(Testdir) of
-        false ->
-            ?INFO("Test directory ~s does not exist:\n",
-                  [Testdir]),
-            ?FAIL;
-        _ -> ok
-    end.
+test_opts(State, InDir, OutDir) ->
+    ErlOpts = rebar_state:get(State, common_test_compile_opts, []) ++
+              rebar_utils:erl_opts(State),
+    TestOpts = [{outdir, OutDir}] ++
+               [{src_dirs, InDir}] ++
+               [{src_dirs, ["src", "test"]}] ++
```

Comment:
maybe. the current code surrounding `src_dirs` could use improving (see [here](https://github.com/rebar/rebar3/blob/master/src/rebar_erlc_compiler.erl#L145) for example) but the two are currently functionally the same

it should probably be changed for future proofing if we decided to change how `src_dirs` is handled tho